Utilities/wasm.cpp

```diff
@@ -264,7 +264,7 @@ static bool post_interpret_hook(Wasm::Configuration&, Wasm::InstructionPointer&
         g_continue = false;
         warnln("Trapped when executing ip={}", ip);
         g_printer->print(instr);
-        warnln("Trap reason: {}", interpreter.trap_reason());
+        warnln("Trap reason: {}", interpreter.trap().format());
```

Comment:
It would appear this file was relying on some transitive includes to get nan and nanf from the C library